src/extension/_locales/en/messages.json

```diff
@@ -372,6 +372,9 @@
     "help_next": {
       "message": "Next tip"
     },
+    "next": {
+      "message": "Next"
```

Comment:
Maybe we should add a `description` for translators that this is about proceeding to the next step or tab, in case a gender needs to be provided.

Reply:
On a side note, I think we can also drop all the `help_*` strings above now.